src/kernel-install/kernel-install

```diff
@@ -87,17 +87,16 @@ KERNEL_IMAGE="$2"
 
 if [[ -f /etc/machine-id ]]; then
     read MACHINE_ID < /etc/machine-id
+else
+    MACHINE_ID="Linux"
 fi
```

Comment:
upper case? that's a bit weird, no? i'd just use lowercase here... it's ultimately case-insensitive, since it's fat, but i think we shouldn't suggest case mattered if we use lower/upper case mixed

Reply:
II saw Microsoft in there which was capitalized so I followed that but I don't mind either way so I'll switch to lowercase.

Reply:
Hmm, so Boot Loader Spec type #2 entries also use the `Linux` spelling, in the `EFI/Linux` path. Maybe stick to `Linux` then and ignore what I said above.